config/jobs/cert-manager/cert-manager-presubmits.yaml

```diff
@@ -524,47 +524,44 @@ presubmits:
         - name: ndots
           value: "1"
 
-  # An E2E test job to allow us to manually trigger the Venafi TPP E2E tests
-  # with the following GitHub comment:
-  #
-  #  /test pull-cert-manager-e2e-v1-21-feature-issuers-venafi-tpp
-  #
-  # See https://github.com/jetstack/cert-manager/issues/3555
-  #
-  - name: pull-cert-manager-e2e-v1-21-feature-issuers-venafi-tpp
-    always_run: false
-    optional: true
+  # Verifies upgrade from the latest published release with both Helm chart and
+  # static manifests. This is an optional test.
+  - name: pull-cert-manager-upgrade-v1-21
+    # Run only when requested.
+    always_run: true
+    optional: false
```

Comment:
It says "run only when requested", I guess the comment does not match `optional: false`?

Reply:
Thanks, that's a good catch!